examples/jenkins-image-sample.groovy

```diff
@@ -0,0 +1,119 @@
+/**
+ * This script does nothing in particular (in fact, it deletes its work),
+ * but is meant to show actual usage of most of the API.
+ */
+
+openshift.withCluster() {
+
+    def saSelector1 = openshift.selector( "serviceaccount" )
+    saSelector1.describe()
+    def saSelector2 = openshift.selector( "sa/builder" )
+    def saSelector3 = openshift.selector( "sa", "builder" )
+    def saSelector4 = openshift.selector( "dc", [ "template" : "mytemplate" ] )
+    
+    def templateSelector = openshift.selector( "template", "mongodb-ephemeral")
+        
+    templateSelector.delete('--ignore-not-found')
+    openshift.delete('all,secret', '-l', 'template=mongodb-ephemeral-template', '--ignore-not-found')
```

Comment:
Instead of
```openshift.delete('all,secret', '-l', 'template=mongodb-ephemeral-template', '--ignore-not-found')```
something like:
```
templateSelector.related('all').delete('--ignored-not-found')
templateSelector.related('secret').delete('--ignored-not-found')
```